Gemfile

```diff
@@ -45,15 +45,14 @@ gem 'ffi-icu'
 gem 'unicode'
 gem 'rack-openid'
 gem 'ruby-openid'
-gem 'secure_headers'
```

Comment:
The idea was to have it as similar as it could with the Un Pais en Comu fork: https://github.com/EnComu/participa/commit/2cd8f7239486a192f972bc8688ea3884a71c23ed